Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: verify-glide-installation install-vendor update-vendor all install check
 
 all:
+	go generate github.com/giantswarm/aws-operator/bindata
```

Comment:
All the examples in https://blog.golang.org/generate call `go generate` explicitly.